email.go

```diff
@@ -50,6 +53,12 @@ type Params struct {
 	InlineImages    []string // InlineImages images path
 }
 
+// List of supported authentication methods
+const (
```

Comment:
my suggestion is to use "fake enum", i.e custom type from string and a pair of values. This way we won't need to pass string as meth

Reply:
Agree